### PR TITLE
modify windows links to include x64 and x32 builds

### PR DESCRIPTION
**Pull request by jmckisson**
We're creating both 32 and 64 bit builds of the Windows app now, this modifies the PR comment to include both.

### CODE REVIEW

**Review by vadi2 — APPROVED** [review]
Thx!

**Comment by vadi2** [comment]
@keneanung could you review this one as well?

File: index.js [hunk 1]
```diff
@@ -96,8 +97,14 @@ const getMudletSnapshotLinksForPr = async prNumber => {
         return -1
       }
     }))
-    // now take the latest link
-    .mapValues(value => value[0])
+    // for windows, take the two latest links: windows-64 and windows-32
+    .mapValues((value, platform) => {
+      if (platform === 'windows') {
+        return value.filter(val => /windows-64|windows-32/.test(val.filename)).slice(0, 2)
```
Comment (keneanung) [inline index.js:103]: These may end up to be for the same architecture in certain cases, but let's wait and see if this can happen at all.

**Review by keneanung — APPROVED** [review]
Sorry for the wait... Let's try this, though we may end up with wrong outdated windows URLs for one architecture and missing ones for the other if one build doesn't work out in a single cycle.

**Comment by keneanung** [comment]
I deployed the change as well. Until all PRs have the new windows line in the comment, Windows links may not be updated.

**Pull request merged**